app/mailers/event_mailer.rb

```diff
@@ -5,15 +5,15 @@ class EventMailer < ApplicationMailer
   before_action :set_emails
 
   def monthly_donation_summary
-    @event = params[:event]
+    @correction = params[:correction]
```

Comment:
I think we need to still set `@event`

Reply:
I noticed that it was already being set with the `before_action` on line 4 - it worked when I tested in development

Reply:
good catch! you're all good here